build-system/pr-check/npm-checks.js

```diff
@@ -17,49 +17,46 @@
 
 /**
  * @fileoverview
- * This script makes sure that package.json and yarn.lock
- * are in sync and up to date.
+ * This script makes sure that package.json and package-lock.json are in sync
+ * and up to date.
  */
 
+const checkDependencies = require('check-dependencies');
 const colors = require('ansi-colors');
-const {getStderr} = require('../common/exec');
 const {gitDiffColor, gitDiffNameOnly} = require('../common/git');
 
 /**
- * Makes sure package.json and yarn.lock are in sync.
+ * Makes sure package.json and package-lock.json are in sync.
  * @param {string} fileName
  * @return {boolean}
  */
-function isYarnLockFileInSync(fileName = 'yarn-checks.js') {
+function isPackageLockFileInSync(fileName = 'npm-checks.js') {
```

Comment:
two nits:
- It doesn't look like there's any scenario in which it should be legal to call this without providing the filename (much like our `timedExec` helpers). It may be reasonable to drop this default altogether, given that it's unused and not intended to be used.
- If you're reluctant to require the file name, since it has a default, I think `opt_fileName` is preferred

Reply:
Nice catch. I've implemented your first suggestion.